bin/sandbox

```diff
@@ -49,13 +49,8 @@ else
   GENERATOR_SKIP_SPECS_ARGUMENT=''
 fi
 
-# Stay away from the bundler env of the containing extension.
-function unbundled {
-  ruby -rbundler -e'b = proc {system *ARGV}; Bundler.respond_to?(:with_unbundled_env) ? Bundler.with_unbundled_env(&b) : Bundler.with_clean_env(&b)' -- $@
-}
```

Comment:
https://makandracards.com/makandra/15649-how-to-discard-a-surrounding-bundler-environment provides a good explanation of the scenario we are in, where we have Gemfiles on a directory and its subdirectory:

![Uploading image.png…]()